src/metabase/sync/util.clj

```diff
@@ -341,11 +339,16 @@
 
 (defn db->reducible-sync-tables
   "Returns a reducible of all the Tables that should go through the sync processes for `database-or-id`."
-  [database-or-id]
-  (t2/reducible-select :model/Table, :db_id (u/the-id database-or-id), {:where sync-tables-clause}))
+  [database-or-id & {:keys [schema-names table-names]}]
+  (t2/reducible-select :model/Table
+                       :db_id (u/the-id database-or-id)
+                       {:where [:and sync-tables-clause
+                                (when (seq schema-names) [:in :schema schema-names])
+                                (when (seq table-names) [:in :name table-names])]}))
 
 (defn db->sync-schemas
-  "Returns all the Schemas that have their metadata sync'd for `database-or-id`."
+  "Returns all the Schemas that have their metadata sync'd for `database-or-id`. If the database doesn't support.
+  Assumes the database supports schemas."
```

Comment:
I'm finding this docstring hard to understand with the second sentence fragment, could you reword it to be clearer?

Reply:
fixed, thanks